app/scripts/lib/metrics.js

```diff
@@ -339,6 +339,8 @@ define([
      */
     logScreen: function (screenName) {
       this.logEvent(this.screenToId(screenName));
+      // log the path as a separate path event
+      this.logEvent('path.' + this._window.location.pathname);
```

Comment:
There is a subtle problem here when using the iframe flow. In the iframe flow, [the pathname is not updated, rather the url's hash is](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/app-start.js#L578-L587).

What does `screenName` have in it at this point? I thought it should be the equivalent of what `window.location.pathname` would be for the standard flow.